tasks/authentication.yml

```diff
@@ -16,12 +16,12 @@
   service:
     name: influxdb
     state: restarted
-  when: influxdb_user_password
+  when: influxdb_user_password != ''
```

Comment:
influxdb_user_password is lalways defined but, it might be null.

Reply:
Ok thank you for spotting that, however I don't think testing for empty string is the same as testing for null, therefore I think that also this variant is wrong.

Reply:
Initially, I also compared the variable with `null` but the CI suite failed for that comparison. 
https://github.com/openwisp/ansible-ow-influxdb/runs/5198855962?check_suite_focus=true#step:7:292 

In CI, we are testing different scenarios:
https://github.com/openwisp/ansible-ow-influxdb/blob/6263005be795ccd0d36b09cfbc993a2f7a56a6ae/.github/workflows/ci.yml#L20-L34

So I think, we should be covered after this change.

Reply:
what about something like: `when: influxdb_user_password | bool`? I would feel more comfortable in a construct like this, which should catch both null and empty string as false conditions.